src/app/zap-templates/templates/app/callback-stub-src.zapt

```diff
@@ -132,3 +132,10 @@ void __attribute__((weak)) MatterPostAttributeChangeCallback(
     uint8_t type, uint16_t size, uint8_t * value)
 {
 }
+
+void __attribute__((weak)) MatterPreExecuteCommandCallback(chip::app::CommandHandler * commandObj,
```

Comment:
We already have a weak callback `MatterPreCommandReceivedCallback` defined that gets invoked before a command gets executed. Why isn't that sufficient?

Reply:
We need to access to the commandObj object to get information about current fabric which execute the command